Makefile

```diff
@@ -171,6 +178,27 @@ testunit: ${BUILD_BIN_PATH}/ginkgo
 	# fixes https://github.com/onsi/ginkgo/issues/518
 	sed -i '2,$${/^mode: atomic/d;}' ${COVERAGE_PATH}/coverprofile
 	$(GO) tool cover -html=${COVERAGE_PATH}/coverprofile -o ${COVERAGE_PATH}/coverage.html
+	$(GO) tool cover -func=${COVERAGE_PATH}/coverprofile | sed -n 's/\(total:\).*\([0-9][0-9].[0-9]\)/\1 \2/p'
+
+testunit-bin:
+	mkdir -p ${TESTBIN_PATH}
+	for PACKAGE in ${PACKAGES}; do \
+		go test $$PACKAGE \
+			--tags "containers_image_ostree_stub $(BUILDTAGS)" \
+			--gcflags '-N' -c -o ${TESTBIN_PATH}/$$(basename $$PACKAGE) ;\
+	done
+
+mockgen: ${BUILD_BIN_PATH}/mockgen
+	${BUILD_BIN_PATH}/mockgen \
```

Comment:
Should we use go:generate comments instead of calling mockgen directly? (We can certainly revisit it.)

Reply:
Yes we could, that would mean that the build targets looks like:

```make
mockgen: ${BUILD_BIN_PATH}/mockgen
	FLAGS=${MOCKGEN_FLAGS} go generate  ./...
```

and then we could collect the generation commands in `framework.go`:

```go
package framework

//go:generate ../../build/bin/mockgen ${FLAGS} -package containerstoragemock -destination=../mocks/containerstorage/containerstorage.go github.com/containers/storage Store
//go:generate ../../build/bin/mockgen ${FLAGS} -package criostoragemock -destination=../mocks/criostorage/criostorage.go github.com/kubernetes-sigs/cri-o/pkg/storage ImageServer
```

From a build time perspective it makes no difference.